actor-testkit-typed/src/main/scala/org/apache/pekko/actor/testkit/typed/internal/StubbedActorContext.scala

```diff
@@ -243,11 +243,17 @@ private[pekko] final class FunctionRef[-T](override val path: ActorPath, send: (
       .iterator()
       .asScala
       .map { evt =>
+        val marker: Option[Marker] =
+          try {
+            Option(evt.getMarker)
+          } catch {
+            case _: NoSuchMethodError => None // evt.getMarker was replaced in slf4j v2 with evt.getMarkers
```

Comment:
Overall I like the direction of the PR but would it be possible to use reflection to get `evt.getMarkers` if it happens to point to the same type as `evt.getMarker` (just contained within a list)?

That way we could get the best of both worlds.

Now that I think of it, honestly think that we likely should have bumped to slf4j in Pekko 2.0.x series since this is strictly breaking but this may be good enough.